src/components/vote/proposals/OpenProposals.vue

```diff
@@ -30,265 +30,256 @@
-              <remaining-time
-                type="warn"
-                :show-seconds="true"
-                :from="proposal.start"
-                :to="proposal.end"
-              />
+          </b-col>
+          <b-col cols="6">
+            <div
+                class="font-size-12 pb-1"
+                :class="darkMode ? 'text-muted-dark' : 'text-muted-light'"
+            >
+              Proposed by
+              <a
+                  target="_blank"
+                  class="font-size-12 font-w500 fix-a"
+                  :href="getEtherscanUrl(item.proposer)"
+              >
+                {{ shortAddress(item.proposer) }}
+              </a>
             </div>
-          </td>
-          <td>
-            <div class="pl-3 container-border h-100">
-              <div
-                v-if="!proposal.votes.voted && proposal.end > Date.now()"
-                class="d-flex align-items-center mb-2"
+            <div
+                class="font-size-12"
+                :class="darkMode ? 'text-muted-dark' : 'text-muted-light'"
+            >
+              Contract to execute
+              <a
+                  target="_blank"
+                  class="font-size-12 font-w500 fix-a"
+                  :href="getEtherscanUrl(item.executor)"
               >
-                <main-button
-                  @click="() => voteFor(proposal.id.toString())"
-                  label="Vote for"
-                  :large="true"
-                  :active="true"
-                  :block="true"
-                  class="font-size-14 font-w400 mr-3 text-uppercase button-vote button-vote--for"
-                />
-
-                <main-button
-                  @click="voteAgainst(proposal.id.toString())"
-                  label="Vote against"
-                  :large="true"
-                  :active="true"
-                  :block="true"
-                  class="font-size-14 font-w400 mt-0 text-uppercase button-vote button-vote--against"
-                />
-              </div>
+                {{ shortAddress(item.executor) }}
+              </a>
+            </div>
+          </b-col>
+        </b-row>
+
+        <div class="pt-2">
+          <remaining-time
+              type="warn"
+              :show-seconds="true"
+              :from="item.start"
+              :to="item.end"
+          />
+        </div>
+      </template>
 
-              <div v-if="proposal.votes.voted">
-                <div
-                  class="votes-bar--empty voted-box mb-2"
-                  :class="'votes-bar--' + proposal.votes.voted"
-                >
-                  <div class="row">
+      <template #cell(votes)="{ item }">
+        <div class="pl-3 container-border h-100">
+          <div
+              v-if="!item.votes.voted && item.end > Date.now()"
+              class="d-flex align-items-center mb-2"
+          >
+            <main-button
+                @click="() => voteFor(item.id.toString())"
```

Comment:
I don't think you wrote this but lets do this one properly. This ain't no React project!

Reply:
fixed